package.json

```diff
@@ -17,6 +17,7 @@
     "postinstall": "yarn typechain && yarn sdk"
   },
   "dependencies": {
+    "@eth-optimism/sdk": "^1.1.7",
```

Comment:
Something wrong happened with the yarn.lock file. It has so many "+/-" changes for a single package.

Reply:
yeah... that's the issue that we discuss the other day. I'll investigate what's going on...

Reply:
I think that's reasonable. The problem here is that we still have a lot of `^` in the `package.json` so each patch done by any of our dependencies OR sub-dependencies will be updated on a new install...
That's another reason (apart from security) why we should keep the `^`to the minimum (only trusted packages)